mrc-1543 dockerise app

hillalex opened the pull request.

EmmaLRussell commented on README.md:
````diff
@@ -14,12 +23,28 @@ Requirements:
+To run:
 
+```
+docker run -p 8080:8080 mrcide/hint:branch_name
````
Replace hint with mint..

EmmaLRussell commented on README.md:
````diff
@@ -14,12 +23,28 @@ Requirements:
+or
 
+```
+docker run -p 8080:8080 mrcide/hint:git_sha
````
..and here too

hillalex commented:
> ` docker run -p 8080:8080 mrcide/mint:mrc-1543` should work shouldn't it?
> 
> I get `docker: Error response from daemon: manifest for mrcide/mint:mrc-1543 not found: manifest unknown: manifest unknown.`
> Has travis not built the image?
> 
> Oh, this is in the log: `"docker push" requires exactly 1 argument`
> https://travis-ci.com/github/mrc-ide/mint/jobs/322060230
> Guess it failed silently?

ah I think it's because it's in the after_success step, so it doesn't fail the build. moved to scripts step now (and fixed the script)

hillalex commented:
Cool looks like it worked now!
